src/components/pages/Navbar/NavbarFeatures/MentorPopover.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const MentorPopover = () => (
+  <>
```

Comment:
I like the explanation pop up--it helps clarify what and why for this added functionality.